src/PrestaShopBundle/Resources/config/services/services.yml

```diff
@@ -325,6 +325,14 @@ services:
 
     prestashop.twig.modules.loader:
         class: 'PrestaShopBundle\Twig\Locator\ModuleTemplateLoader'
-        arguments: ['@=service("prestashop.module_kernel.repository").getActiveModulesPaths()']
+        arguments:
+          - '@=service("prestashop.module_kernel.repository").getActiveModulesPaths()'
+          - ~
+          -
+            'PrestaShop': ''
+            'Product': '/Admin/Product'
+            'Twig': '/Admin/TwigTemplateForm'
+            'AdvancedParameters': '/Admin/Configure/AdvancedParameters'
+            'ShopParameters': '/Admin/Configure/ShopParameters'
```

Comment:
Why not store this list elsewhere, like  in a config or something?

Reply:
It use just once, not sure it's necessary

Reply:
because this won't be re-used.

At start, I wanted to re-use the paths used in framework configuration, but thinking about it it's not a good idea. The 'array' look like the same but they have really different purposes.

If we have another place/service when we want to re-use it, of course I'm also in favor of extracting it into a parameter.

Note you can access to all registered Twig paths and namespaces using `php bin/console debug:twig ` but not us, cause of weird call of Legacy Context in a Twig extension. This is a really minor issue we should try to fix asap.

Reply:
But we'll need to add new namespaces as we migrate new sections, won't we?

Reply:
yes, so we will append new namespaces here.

But before I need to document which namespaces should be added as there is no real consistency actually: some templates use them, others don't, some namespace refers to a page, others to sections or even categories...

Reply:
As long as it isn't done in runtime...

Reply:
![array](https://user-images.githubusercontent.com/1247388/41193648-637ac042-6c0f-11e8-864f-441bfe542e5f.png)
nope, it's converted in PHP function in container cache: I don't think we can make the process fast enough :+1:

Reply:
yes everything setting and yml files are automatically converted in string / array in the cache file =)